### PR TITLE
Update example_transactions.md

### PR DESCRIPTION
**Pull request by whitemagpie**
changed a few outputs mainly coz the error messages and details in outputs have changed with new version of Bitcoin Core.

### CODE REVIEW

File: _includes/devdoc/example_transactions.md [hunk 1]
```diff
@@ -768,6 +768,15 @@ subsections.
...
+        "errors": [
+    {
+      "txid": "d581f4363640aaa978e13179983da38965ab9aa33855769ec2f2cddf55f1b8e5",
+      "vout": 1,
+      "scriptSig": "",
+      "sequence": 4294967295,
+      "error": "Input not found or already spent"
+    }
+  ]
     }
```
Comment (Mirobit) [inline _includes/devdoc/example_transactions.md]: There are some whitespaces missing (772-779).

File: _includes/devdoc/example_transactions.md [hunk 2]
```diff
@@ -840,7 +849,9 @@ broadcasts it.
 > bitcoin-cli -regtest sendrawtransaction $SIGNED_RAW_TX
 {% endhighlight %}
 {% highlight json %}
-error: {"code":-22,"message":"TX rejected"}
+error code: -25
+error message:
+Missing inputs
 {% endhighlight %}
```
Comment (Mirobit) [inline _includes/devdoc/example_transactions.md]: Is this the original formatting of the error message? Mine is structured like this 

> 258: txn-mempool-conflict (code -26)

File: _includes/devdoc/example_transactions.md [hunk 3]
```diff
@@ -934,19 +945,22 @@ redeem script. You must give them a full public key.
...
 {
-    "isvalid" : true,
-    "address" : "mk2QpYatsKicvFVuTAQLBryyccRXMUaGHP",
-    "ismine" : true,
-    "isscript" : false,
-    "pubkey" : "029e03a901b85534ff1e92c43c74431f7ce72046060fcf7a\
-                95c37e148f78c77255",
-    "iscompressed" : true,
-    "account" : ""
+   "isvalid": true,
```
Comment (Mirobit) [inline _includes/devdoc/example_transactions.md]: One space to much.

**Comment by wbnns** [comment]
@whitemagpie Hello, thanks for submitting this. Could you please incorporate @Mirobit 's suggestions and also rebase this PR to resolve the conflicts?

@Mirobit Thanks for the review!

**Comment by wbnns** [comment]
@whitemagpie Hello, since it's been a while, I'm going to go ahead and close this for now. If you have time in the future to incorporate the changes that @Mirobit mentioned, it would be great if you could submit a new pull request!

**Pull request closed without merging**